start-from-zero/Javascript/src/data/data.js

```diff
@@ -1,4 +1,4 @@
-function InputData(){
+windowObjectWrap.InputData = function(){
```

Comment:
While this solution is one step closer to our eventual goal, it still exposes what is likely more functionality than we need to expose. Are all of these truly worth exposing as publicly available methods?

Reply:
No, they're not. I will try to implement what you presented us in the morning. :)